### PR TITLE
Add slackin engllish translation module prototype, translation tests & ...

### PR DESCRIPTION
**Pull request by vrunoa**
...spanish, deutsche translations

### CODE REVIEW

**Comment by rauchg** [comment]
@vrunoa super interested in this. We should clean it up and merge it <3

**Comment by vrunoa** [comment]
hey @rauchg i'll clean it up, update my repo which is quite behind master and let you know.

**Comment by vrunoa** [comment]
@rauchg I merged the repo and rewrite a bit of the localization code, let me know if you're OK with this implementation or want to change something.

**Comment by KrzysztofMadejski** [comment]
I'll add Polish translations when this pull request will go live. Thanks for the work!

File: bin/slackin [hunk 1]
```diff
@@ -16,6 +16,7 @@ program
 .option('-s, --silent', 'Do not print out warns or errors')
 .option('-C, --coc <coc>', 'Full URL to a CoC that needs to be agreed to')
 .option('-c, --css <file>', 'Full URL to a custom CSS file to use on the main page')
+.option('-l, --locale [<locale>]', 'One or more comma-separated desired localization translations to serve', 'en')
```
Comment (cvan) [inline bin/slackin]: have you looked at using [L20n](http://l20n.org/)?

**Pull request closed without merging**